packages/did-ipfs-store/src/didStore.ts

```diff
@@ -1,46 +1,57 @@
-/* eslint-disable no-restricted-syntax */
-import ipfsClient from 'ipfs-http-client';
-import BufferList from 'bl';
+import { Cluster } from '@nftstorage/ipfs-cluster';
 import { IDidStore } from '@ew-did-registry/did-store-interface';
+import fetch from '@web-std/fetch';
+import { FormData } from '@web-std/form-data';
+import { File, Blob } from '@web-std/file';
+import axios from 'axios';
+
+Object.assign(global, { fetch, File, Blob, FormData });
 
 export class DidStore implements IDidStore {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  private ipfs: any;
+  private cluster: Cluster | undefined;
 
   /**
-   * @param uri {string} - IPFS API server uri
+   * @param url Ipfs cluster root
+   * @param headers Http request headers
    */
-  constructor(uri: string | object) {
-    this.ipfs = ipfsClient(uri);
-  }
+  constructor(
+    private url: string,
+    private headers: Record<string, string> = {}
+  ) {}
 
```

Comment:
@JGiter Can you please add some method documentation here?

Reply:
I can add more info about class itself. Public methods are documented in implemented interface

Reply:
Or do you think there something more can be said?

Reply:
Actually `get` method had some documentation while `save` and `delete` didn't have, so it could be consistent if we added some documentation for these two as well.

Reply:
Agree on consistency. Moved method documentation to method definitions